research/mark_generator/00_training.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        base_model.trainable = False
</pre><p>Have you tried training entire model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cvlab-ai/splinter/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>